app/views/works/_show_info.html.erb

```diff
@@ -184,11 +184,8 @@
   <p class="citation">
      <%=CitationDisplay.new(@work).display %>
   </p>
-
-
-  <%#
   <p class="btn-group" role="group">
-    <a class="btn btn-sm btn-primary export-citation" href="/works/< % = @work.friendlier_id % >.ris">
+    <a class="btn btn-sm btn-primary export-citation" href="/works/<%= @work.friendlier_id %>.ris">
```

Comment:
`href="/works/<%= @work.friendlier_id %>.ris"` is the wrong way to do this. We want to use Rails routing helpers!

To generate the URL: 

    work_path(@work, format: :ris)

To generate the link nicely without having to put ERB tags in an attribute, let's use Rails `link_to`. 

    link_to(work_path(@work, format: :ris), class: "btn btn-sm btn-primary export-citation") do
       <i class="fa fa-address-card"></i>&nbsp;&nbsp;Export citation (RIS)
     end

Just typed that in without trying it, may have typos or errors, but that's the idea.